docs/howto/openstack/magnum/kubectl.md

```diff
@@ -0,0 +1,91 @@
+---
+description: How to fetch, verify, and use your kubeconfig with kubectl in a Magnum-managed Kubernetes cluster.
+---
+# Managing a Kubernetes cluster
+
+Once you [have launched a new cluster](new-k8s-cluster.md), you can interact with it using `kubectl` and a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file.
+
+## Prerequisites
+
+You must install the Kubernetes command line tool, `kubectl`, on your local computer, and run commands against your cluster.
+To install `kubectl`, follow [the relevant Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).
+
+## Extracting the kubeconfig file
+
+Due to Magnum's security policy configuration, you cannot use the OpenStack CLI for downloading the kubeconfig of a cluster that was created with {{gui}}, or vice versa.
+
+To fetch your kubeconfig, you must always use the same facility that you used to deploy the cluster.
+
+=== "{{gui}}"
+    In the left-hand side pane of the {{gui}}, select *Magnum* → *Clusters*.
+    Click on the cluster row to expand the details view, then click the
+    *KubeConfig* tab. In a second or two, you will see the contents of the
+    kubeconfig file. Click the blue *Download KubeConfig* button to download
+    it locally.
+
+    ![Kubeconfig view](assets/shot-07.png)
+
+    The kubeconfig file you get has a name similar to this one:
+
+    ```plain
+    kubeconfig--<cluster_name>--<region_name>--<alphanum_id>.yaml
+    ```
+
+    Feel free to rename it to something simpler, like `config`.
+=== "OpenStack CLI"
+    To download the kubeconfig file for your Kubernetes cluster, type
+    the following:
+
+    ```bash
+    openstack coe cluster config --dir=${PWD} bangor
```

Comment:
Since Bangor is not on our radar in this HOWTO, I suggest this change:

```suggestion
    openstack coe cluster config --dir=${PWD} <cluster_name>
```

Reply:
Ah yes, absolutely. Will fix.